src/utils/src/types/risk-types.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { FromSchema } from 'json-schema-to-ts';
+import { bodySchema, intType, stringType, booleanType } from './api-utils-types';
+
+export const riskEditInputSchemaBody = bodySchema({
+  userId: intType,
+  id: intType,
```

Comment:
if you look at the schema.prisma, you can see we actually use a string id for risks (which we should be doing everywhere but woops lol) so this should be stringType. This will also fix the compile errors in risk-edit.ts that you're getting